examples/simple/src/comments/PostPreview.js

```diff
@@ -11,7 +11,6 @@ const PostPreview = props => {
         [props.resource, props.id]
     );
     const version = useSelector(state => state.admin.ui.viewVersion);
-    const isLoading = useSelector(state => state.admin.loading > 0);
```

Comment:
Why did you remove this one? It forces a redraw when isLoading changes.

Reply:
Right, it complained about the unused variable